### PR TITLE
First exercise

### PR DESCRIPTION
**Pull request by LironAttalGit**

### CODE REVIEW

File: src/main/java/consts/Const.java [hunk 1]
```diff
@@ -1,6 +1,14 @@
 package consts;
 
+import java.util.HashMap;
+
 public class Const {
-    public static final String encrypt = "1";
-    public static final String decrypt = "2";
-}
+
+    public static HashMap<String, String> options = new HashMap<>();
```
Comment (Aviv454) [inline src/main/java/consts/Const.java:7]: I like where u went with the options, i think an enum would be a better option here

File: src/main/java/dataManipulation/DataManipulation.java [hunk 2]
```diff
@@ -1,9 +1,12 @@
 package dataManipulation;
 
-import java.util.Arrays;
+import userInput.UserInput;
+
 import java.util.Random;
```
Comment (Aviv454) [inline src/main/java/dataManipulation/DataManipulation.java:6]: this comment is for the entire file - great separation!

File: src/main/java/encryptor/Encryptor.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    UserInput userInput = new UserInput();
+    FileManager fileManager = new FileManager();
+
+    DataManipulation dataManipulation = new DataManipulation();
+
+    public void start() {
+        System.out.println("starting encryptor");
+        Validators validator = new Validators();
+        String type = validator.verifyAction();
+        if(! type.equals(options.get("QUIT"))) {
```
Comment (Aviv454) [inline src/main/java/encryptor/Encryptor.java:27]: in continuation to the const class, i would combine the quit and the options at this point

File: src/main/java/validators/Validators.java [hunk 4]
```diff
@@ -1,39 +1,37 @@
...
+        String inp;
         do {
-            if(type.equals("1") || type.equals("2")) {
-                break;
-            }
-            System.out.println("Please choose 1 for encrypt or  2 for decrypt");
-            type = scanner.nextLine();
-        } while(! type.equals("1") || ! type.equals("2"));
-        return type;
+            options.forEach((key , value) ->
```
Comment (Aviv454) [inline src/main/java/validators/Validators.java:16]: nice implemetation, but forEach iterates over every possible option, which can be quite inefficient

**Review by Aviv454 — COMMENTED** [review]
the separation and shortage of code looks great! 
fix the 2 comments and keep on to the next chapter.

File: src/main/java/validators/Validators.java (hunk 4 above)
Comment (LironAttalGit) [reply src/main/java/validators/Validators.java:16]: true, but i want to print all the options

**Pull request closed without merging**